tests/Test/AbstractFixerTestCase.php

```diff
@@ -532,7 +532,7 @@ final public function testProperMethodNaming(): void
             YodaStyleFixerTest::class,
         ];
 
-        $names = ['Fix', 'Fix74Deprecated', 'FixPre80', 'Fix80', 'FixPre81', 'Fix81', 'Fix82', 'Fix83', 'FixPre84', 'Fix84', 'WithWhitespacesConfig', 'InvalidConfiguration'];
+        $names = ['Fix', 'Fix74Deprecated', 'FixPre80', 'Fix80', 'FixPre81', 'Fix81', 'Fix82', 'Fix83', 'FixPre84', 'Fix84', 'FixUnit11', 'WithWhitespacesConfig', 'InvalidConfiguration'];
```

Comment:
This change is not necessary. We have strict naming convention and it's related to PHP features, not to PHPUnit version. As I wrote in the other comment - test cases added in this PR should be added to already existing `provideFix80Cases`.